library/ssl_tls.c

```diff
@@ -3477,6 +3479,8 @@ static unsigned char ssl_serialized_session_header[] = {
  *
  *      case MBEDTLS_SSL_VERSION_TLS1_2:
  *        serialized_session_tls12 data;
+ *      case MBEDTLS_SSL_MINOR_VERSION_4:
```

Comment:
Should this be `MBEDTLS_SSL_VERSION_TLS1_3` ?
Should we add the following line under line 3476?
```C
 *                         // - TLS 1.2 (3)
```